app/models/outgoing_email.rb

```diff
@@ -38,6 +38,7 @@ class OutgoingEmail < ApplicationRecord
   validates_presence_of :body
   validates_presence_of :subject
   validates :mailgun_status, inclusion: { in: ALL_KNOWN_MAILGUN_STATUSES }
+  attr_accessor :locale, :tax_return
```

Comment:
I don't understand the purpose of this yet.

Reply:
oh good catch. it is unneccessary, a remnant of when i was going to process replacement params on the model instead of in the service.